### ISSUE TITLE
create a model.sensePart class and change content model of <sense> accordingly

### ISSUE THREAD

**Issue by laurentromary** [TEIC/TEI#1809]
The content models of `<entry>` and `<sense>` are doomed to being aligned because they both use model.entryPart.top. This is a real issue when ne wants to customize one or the other elements and more generally, it does not make real 'sense' to have `<sense>` in the same group as `<entry>`, `<hom>` and `<re>`. A simple correction for this would be to create a model.sensePart class, and ensure a smooth transition by making all current members of model.entryPart.top members of this class, adding `<sense>` itself, in order to make the element recursive automatically (and thus simplify the current content model). This proposal is not changing anything from a behavioral point of view, but would allow, for instance, the TEI lex 0 group to fine tune the content model of `<entry>` without impacting on `<sense>`.

**Comment by scstanley7** [comment]
Many apologies for the delay in addressing this! Just to clarify: the proposed model.sensePart would contain `<entry>`, `<hom>`, `<re>` *and* `<sense>`?

How do you feel about the following language in "remarks" for this new model class? 
> groups elements belonging to the microstructures of dictionary entries

Thanks for your patience.

**Comment by laurentromary** [comment]
No, they would not. We suggest it contains: sense cit def dictScrap etym form gramGrp lbl re usg xr (the previous members of model.entryPart.top to which we add sense. The result is pure continuity from the resulting content model.

**Comment by martinascholger** [comment]
Council agrees at F2F
